lib/fixup.rb

```diff
@@ -160,24 +160,24 @@ def refresh_crew_json
   { pkg_name: 'py3_pptx', pkg_rename: 'py3_python_pptx', pkg_deprecated: nil, comments: 'Fix to match upstream name.' },
   { pkg_name: 'pygobject', pkg_rename: 'py3_pygobject', pkg_deprecated: nil, comments: 'Renamed to use pip packaging.' },
   { pkg_name: 'q', pkg_rename: nil, pkg_deprecated: true, comments: 'No longer suitable for packaging.' },
-  { pkg_name: 'qtbase', pkg_rename: 'qt5_base', pkg_deprecated: nil, comments: 'Qt packages renamed to qt5_*' },
-  { pkg_name: 'qtcharts', pkg_rename: 'qt5_charts', pkg_deprecated: nil, comments: nil },
+  { pkg_name: 'qtbase', pkg_rename: 'qt5_base', pkg_deprecated: nil, comments: 'Renamed to allow qt6 packages.' },
+  { pkg_name: 'qtcharts', pkg_rename: 'qt5_charts', pkg_deprecated: nil, comments: 'Renamed to allow qt6 packages.' },
```

Comment:
I actually think renaming them back to qt5_* ALLOWS for qt6 packages.  Once we have both, we'll need to distinguish between them.

Reply:
Exactly-- I figured specifying that here would make things more clear to users.

Reply:
The only problem is users won't know to look here to find that out.

Reply:
These comments are printed when the user updates from an older version before the rename, so it will be made known to users that it applies to.